entrypoint.sh

```diff
@@ -4,9 +4,19 @@ set -e
 zulip_organization_url=$1
 zulip_bot_email=$2
 zulip_bot_api_key=$3
-github_personal_access_token=$4
+github_token=$4
 delete_history=$5
 archive_branch=$6
+github_personal_access_token=$7
+
+github_personal_access_token=${github_personal_access_token:-NOT_SET}
+
+if [ $github_personal_access_token != "NOT_SET" ]; then
+    echo "'github_personal_access_token' input has been deprecated."
+    echo "Please use 'github_token' instead."
```

Comment:
The instruction should be "Please remove its usage. We have replaced it with 'github_token', which is auto-generated and doesn't require setup."

Reply:
@refeed do you have time to handle this?

Reply:
Can we have that second line be a link to instructions for how you update your installation? That seems cleanest.

Reply:
I made #95. The instruction to update the installation is to simply remove `github_personal_access_token: ${{ secrets.gh_personal_access_token }}` from the user's GH Actions yml, and so I wrote exactly that in the description.

Reply:
It looks like it still needs to be specified explicitly. I'm updating the message then.
```
gh_token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Updated.